packages/neon_framework/lib/src/storage/sqlite_cookie_persistence.dart

```diff
@@ -95,12 +95,13 @@ CREATE TABLE "cookies" (
 );
 
 CREATE TRIGGER delete_outdated_cookies
-   BEFORE UPDATE
+   AFTER UPDATE
    ON cookies
 BEGIN
  DELETE FROM cookies
  WHERE
-   "expiry-time" <= STRFTIME('%s');
+   "expiry-time" <= STRFTIME('%s')
+   OR "last-access-time" <= STRFTIME('%s', 'now', '-1 years');
```

Comment:
The time range could be pulled out into a separate variable to avoid mismatches if it would be changed later on.

Reply:
I can't come up with a good variable name.
I was thinking of `_lastAccessCheck`; do you have a better suggestion?

Reply:
Maybe `_lastAccessMaxDuration`?